notebooks/examples/CEMS.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>You can also do <code>dd.dtypes</code>  to get a list of both the column names, and their data types, which is a bit more informative.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>better, agreed.</p>